Packs/Coralogix/Integrations/README.md

```diff
@@ -0,0 +1,1710 @@
+## Overview
+---
+
+Automatically fetch security incidents identified by Coralogix
+This integration was integrated and tested with Logz.io platform.
+## Logz.io Playbook
+---
+Logz.Io Handle Alert: used to handle alerts retrieved from Logz.io. The playbook will retrieve the related events that generated the alert using the logzio-get-logs-by-event-id command
+
+## Use Cases
+---
+
+Integrate with Logz.io Cloud SIEM to automatically remediate security incidents identified by Logz.io and increase observability into incident details. 
+The integration allows Cortex XSOAR users to automatically remediate incidents identified by Logz.io Cloud SIEM using Cortex XSOAR Playbooks.
+In addition, users can query Logz.io directly from Cortex XSOAR to investigate open questions or retrieve the logs responsible for triggering security rules.
+
+## Configure Logz.io on Cortex XSOAR
```

Comment:
What is Logz.io?
It does not look like the correct README

Reply:
Removed this file. You're correct - it's not the right one.